src/integration/common/header.js

```diff
@@ -15,15 +15,31 @@ export default () => {
       expect(langCode).toBeInTheDocument();
     });
 
-    it('I can see the navigation', () => {
-      const navigationItemATag = document.querySelector(
+    describe('Navigation link', () => {
+      const navigationLinks = document.querySelectorAll(
```

Comment:
if you wanted to test the first nav you could do 

```js
const navigationLinks = document.querySelector('header nav [role="list"]').querySelectorAll('a')
```

then you won't get all the duplicate snapshots

Reply:
Worked! Thanks 👍